boto/ec2/connection.py

```diff
@@ -445,6 +452,10 @@ def create_image(self, instance_id, name,
             params['NoReboot'] = 'true'
         if block_device_mapping:
             block_device_mapping.ec2_build_list_params(params)
+        if image_tags:
+            params.update(tags_to_tag_specification(image_tags, 'image'))
+        if snapshot_tags:
+            params.update(tags_to_tag_specification(snapshot_tags, 'snapshot', specification_id=1))
```

Comment:
Давай будем менять id только если 0 будет занято
```suggestion
            params.update(tags_to_tag_specification(
                snapshot_tags, 'snapshot', specification_id=1 if image_tags else 0)
            )
```